webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/index.js

```diff
@@ -94,17 +94,22 @@ const RegistrationCommandsPage = () => {
   // Plugins
   const [pluginValues, setPluginValues] = useState({});
 
-  const handlePluginValue = data => {
-    setPluginValues({ ...pluginValues, ...data });
-  };
+  const handlePluginValue = useCallback(data => {
+    setPluginValues(prevValues => ({ ...prevValues, ...data }));
```

Comment:
The function form of `setState` is great for when your new state value depends on the previous state. But in this case, it's also great because it avoids having to add `pluginValues` to the dependency array, thus avoiding another infinite loop.